src/sentry/tasks/files.py

```diff
@@ -92,6 +97,7 @@ def delete_unreferenced_blobs(blob_model, blob_index_model, blob_ids):
                 pass
 
 
+# TODO(hybrid-cloud): Remove this once backfills are done?
```

Comment:
I think the time for this to be removed is soon. I plan on going through avatar code this week so we can close of that pile of work.

Reply:
Sounds good. I have a local commit that has started removing some of the fallback file lookup as part of the split db work but I can throw it on the backburner it if you're planning to tackle this since I think you have more context that I'm probably missing 😅 .

Reply:
I had some time today to go through avatars again and put up #54118 to make writes use controlfile consistently but still have read support for region files (for single-tenant and self-hosted).